content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+date: 2019-11-13T11:23:04-06:00
+draft: false
+meta_desc: "Pulumi supports .NET Core languages for Infrastructure as Code"
+meta_image: "meta.png"
+authors:
+    - sophia-parafina
+tags:
+    - ".Net Core"
+    - "C#"
+    - "F#"
+    - "Visual Basic"
+---
+
+We are excited to announce the preview of Pulumi .NET support. Pulumi lets you declare infrastructure using general purpose programming languages. Our goal is to automate applications at all layers and help developers and operators work better together by eliminating the silos that separate them.
```

Comment:
We already announced this support before?

Reply:
Right - we posted https://www.pulumi.com/blog/pulumi-dotnet-core/ yesterday as an "announce".  I thought this post was focused on being more of a follow-up walkthrough of the experience using it?  Like taking https://www.pulumi.com/docs/intro/languages/dotnet/ and blowing it out into a few more pages of content?